metricbeat/docs/modules/ibmmq.asciidoc

```diff
@@ -0,0 +1,58 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-ibmmq]]
+[role="xpack"]
+== IBM MQ module
+
+beta[]
+
+This module periodically fetches metrics from IBM MQ.
```

Comment:
We should probably note that this Module only supports IBM MQ containerised version at this very first point too?

Reply:
Fixed.